library/default-synths-extra.scd

```diff
@@ -480,13 +481,13 @@ SynthDef(\superwavemechanics, {|out, pan, freq, sustain, accelerate, detune=0, v
 	var env, sound, i, r, d, bwr, sz;
 	freq = freq * XLine.ar(1, exp(accelerate), sustain);
 	env = EnvGen.ar(Env.sine(1.0), timeScale:sustain, doneAction:2);
-	i = (0..7).scramble;
+	i = Array.fill(8, {TIRand.kr(0, 3, Impulse.kr(8))});
```

Comment:
why trigger at 8 Hz? Also, if `i` is used in line 487 in a `Rand` it will never be retriggered.